jme3-scenecomposer/src/com/jme3/gde/scenecomposer/SceneComposerTopComponent.java

```diff
@@ -910,42 +933,30 @@ private void transformationTypeComboBoxActionPerformed(java.awt.event.ActionEven
         toolController.setTransformationType(transformationTypeComboBox.getItemAt(transformationTypeComboBox.getSelectedIndex()));
     }//GEN-LAST:event_transformationTypeComboBoxActionPerformed
 
-    private void jSlider1StateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_jSlider1StateChanged
```

Comment:
This section confused me for a while, until I realized it was just Netbeans that had decided to shuffle things around